src/verilog/verilog_synthesis.cpp

```diff
@@ -112,6 +111,32 @@ exprt verilog_synthesist::synth_expr(exprt expr, symbol_statet symbol_state)
 
 /*******************************************************************\
 
+Function: verilog_synthesist::function_locality
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void verilog_synthesist::function_locality(const symbolt &function_symbol)
+{
+  // Find all local symbols of the function, mark their
+  // assignments as local.
+  auto prefix = id2string(function_symbol.name) + '.';
+  for(auto &symbol : symbol_table.symbols)
+  {
+    if(irep_idt(symbol.first).starts_with(prefix))
```

Comment:
Should we perhaps get the latest version of the CBMC dependency merged first so that this `irep_idt(...)` wrapper can be avoided?

Reply:
#308